src/bp-activity/bp-activity-notifications.php

```diff
@@ -246,6 +246,18 @@ function bp_activity_update_reply_add_notification( $activity, $comment_id, $com
 		}
 	}
 
+	// Try to find mentions.
+	$comment_activity = new BP_Activity_Activity( $comment_id );
+	$usernames = bp_activity_find_mentions( $comment_activity->content );
```

Comment:
```suggestion
	$usernames = bp_activity_do_mentions() ? bp_activity_find_mentions( $params['content'] ) : array();
```
@rajinsharwar2 We can remove the above code since the filter already provides a fourth parameter, $params, which contains the activity content. So, there's no need to fetch the content again using the BP_Activity_Activity class.